tests/sles4sap/saptune/mr_test.pm

```diff
@@ -276,8 +297,12 @@ sub run {
         # be tested with an override file
         $self->test_note($test) if ($test ne "1805750");
         $self->test_override($test);
-    } elsif ($test =~ m/^(x86_64|ppc64le)$/) {
```

Comment:
Why not keep this and then do:

```
$self->test_x86_64 if (check_var('BACKEND', 'ipmi'));
$self->test_ppc64le if (get_var('OFW'));
$self->text_bsc1152598;
```
That was there's only one call to `$self->text_bsc1152598` in the code.

Reply:
Done.